src/modules/assignment-4/questions/question4/Task4.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from "react";
+import "../Styles.css";
+
+const Task4 = () => {
+  const [formData, setFormData] = useState({});
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const { password, confirmPassword } = formData;
+
+    // Password Validation
+    if (password !== confirmPassword) {
```

Comment:
![image](https://github.com/ayush1-successive/react-training/assets/83696224/474f5f65-4c18-4d8d-86a8-965216592ad0)
align the input fields